get_ns.py

```diff
@@ -0,0 +1,37 @@
+from os import listdir
+from os.path import isfile, join
+path = "/net/u/1/j/jongiles/scripts/domains/names"
+test_string1 = "Name server"
+test_string2 = "name server"
+test_string3 = "Name Server"
+
+active_domains = []
+inactive_domains = []
+
+domainfiles = [f for f in listdir(path) if isfile(join(path, f))]
+##	print (domainfiles)
+for item in domainfiles:
+	no_nameserver = 'False'
```

Comment:
You should be using actual boolean values here, not strings saying `False` and `True`.  :-o